application/src/main/java/org/thingsboard/server/service/security/auth/DefaultTokenOutdatingService.java

```diff
@@ -43,24 +42,22 @@ public DefaultTokenOutdatingService(@Qualifier("UsersSessionInvalidation") TbTra
 
     @EventListener(classes = UserAuthDataChangedEvent.class)
     public void onUserAuthDataChanged(UserAuthDataChangedEvent event) {
-        if (StringUtils.hasText(event.getId())) {
```

Comment:
Let's leave the  ```if (StringUtils.hasText(event.getId())) ```. All other changes are LGTM

Reply:
done